File: Difficulty: Basic/Table Difference/table-difference.py
#{ 
 # Driver Code Starts
#Initial Template for Python 3

# } Driver Code Ends
#User function Template for python3

# Taking input for two numbers
n1 = int(input())
n2 = int(input())

# List to store the differences
difference = []

# Loop to calculate differences for the first 10 multiples
for i in range(1, 11):
    diff = (n1 * i) - (n2 * i)
    difference.append(str(diff))

# Printing the differences space-separated
print(' '.join(difference))


#{ 
 # Driver Code Starts.

# } Driver Code Ends

File: Difficulty: Basic/Table Difference/table-difference.java
//{ Driver Code Starts
//Initial Template for Java

import java.util.Scanner;

class Main {

    
// } Driver Code Ends

// User function Template for Java

public static void difference(int n1, int n2) {
    // Loop through numbers 1 to 10 and print the difference of their multiples
    for (int i = 1; i <= 10; i++) {
        System.out.print((n1 * i) - (n2 * i) + " ");
    }
}


//{ Driver Code Starts.

    public static void main(String[] args)
    {
        Scanner scn = new Scanner(System.in);
        int t = scn.nextInt();
        while(t-- > 0) {
            int n1 = scn.nextInt();
            int n2 = scn.nextInt();
            difference(n1, n2);
            System.out.println();
        
System.out.println("~");
}
        scn.close();
    }
}
// } Driver Code Ends